src/HW/client/src/components/List/List.jsx

```diff
@@ -0,0 +1,30 @@
+import ListItem from "../ListItem/ListItem.jsx";
+import "./List.css";
+import PropTypes from "prop-types";
+const List = ({ items, deleteItemFromDb, updateStatusDb, editTaskNameDb }) => {
+  return (
+    <div>
+      <ul className="tasks">
+        {items.map((item, index) => {
+          return (
+            <ListItem
+              item={item}
+              deleteItemFromDb={deleteItemFromDb}
+              updateStatusDb={updateStatusDb}
+              key={index}
```

Comment:
better to uniq id for the key and not index in the list as the index can change
you can read more in here - 
https://reactjs.org/docs/lists-and-keys.html